source/game/system/RaceManager.cc

```diff
@@ -0,0 +1,47 @@
+#include "game/system/RaceManager.hh"
+
+#include "game/system/CourseMap.hh"
+#include "game/system/map/MapdataStartPoint.hh"
+
+namespace System {
+
+RaceManager *RaceManager::CreateInstance() {
+    assert(!s_instance);
+    s_instance = new RaceManager;
+    return s_instance;
+}
+
+RaceManager *RaceManager::Instance() {
+    return s_instance;
+}
+
+void RaceManager::DestroyInstance() {
+    assert(s_instance);
+    delete s_instance;
+    s_instance = nullptr;
+}
+
+RaceManager::RaceManager() = default;
+
+RaceManager::~RaceManager() = default;
+
+void RaceManager::findKartStartPoint(EGG::Vector3f &pos, EGG::Vector3f &angles) {
```

Comment:
Ordering.

Reply:
Can you clarify what you're suggesting to do here

Reply:
The ordering of these functions are inconsistent with that of the header.